README.MD

```diff
@@ -29,6 +29,7 @@ There may be more to come, time will tell.
 - Includes batch scripts to run MDRP when your music player is started, and will close MDRP when your player closes
 - Can toggle on or off certain music players
 - Can change the default name of the application in Discord, so you could have `Playing Music` instead of `Playing Groove Music`
+- Ability to add your own music player support, assuming the player outputs to WMC properly
```

Comment:
If this is the first reference to `WMC` in the readme, then spell it out. or resolve this. Idk, hard to see in this gh viewport

Reply:
It's mentioned previously on line 21, line 79, and on line 87.